llm-guard/app/scanner.py

```
@@ -0,0 +1,55 @@
+            model=config.MODEL,
+            device=-1,
+            truncation=True,
+            max_length=512,
+            top_k=None,
+        )
+        logger.info("model ready")
+
+    def scan(self, text: str) -> tuple[bool, float]:
+        """Return (is_safe, injection_score) for the given text."""
+        if self._pipe is None:
+            raise RuntimeError("Scanner not loaded; call load() first")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)